addons/mail/static/src/utils/test_utils.js

```diff
@@ -453,6 +453,13 @@ function getCreateComposerSuggestionComponent({ components, env, modelManager, w
     };
 }
 
+function getFileUploaders({ env }) {
+    return async function FileUploaders() {
+        const messaging = await env.services.messaging.get();
+        return messaging.models['FileUploader'].all();
+    };
+}
```

Comment:
I'm not fond of this. I prefer in test using `env.models['FileUploader'].all()` explicitly

Reply:
I don't think this logic should be explicit in the tests since the way we get `FileUploader` could change in the future. 
When it changes, you will need to update all tests and that is always painfull.

I don't see the point of having it explicitly inside the test since we can abstract the "magic" inside a not too badly named function. If it's not good enough, we could try to improve the name.